sormas-api/src/main/java/de/symeda/sormas/api/sormastosormas/SormasToSormasShareInfoDto.java

```diff
@@ -23,7 +23,7 @@
 
 	private static final long serialVersionUID = -1478467237560439811L;
 
-	private ServerAccessDataReferenceDto target;
+	private SormasServerDescriptor targetServerDescriptor;
```

Comment:
Use organization to have it consistent with SormasToSormasShareInfo and add a comment that this is the target server

Reply:
changed for now, java doc added